migration/matchPopulation/purposeIds

```diff
@@ -0,0 +1,3 @@
+'fb885dcf-8639-44a7-97c4-0383a4ebd75c'
+'0caad95b-37c1-47fd-9e47-ac2b96fd70dd'
+'d9a17040-cd35-4983-a9a1-fc080a691bb0'
```

Comment:
I think maybe my earlier comments were not clear. We don't need to check in any id files - it should be something provided by the user of the script, not part of the code base. In other words, calling the script should look something like this:

```
groovy reprocessMatches.groovy \ 
    `gcloud auth print-access-token` \
    https://local.broadinstitute.org:27443 \
    <file path goes here as an argument>
```

Reply:
This file should be removed. The ids are going to be specific to the environment.